models/user.js

```diff
@@ -20,6 +20,7 @@ const UserSchema = new mongoose.Schema({
   show_username: { type: Boolean, default: false },
   show_volume_traded: { type: Boolean, default: false },
   lightning_address: { type: String },
+  nostrPublicKey: { type: String },
```

Comment:
why users need to add their pubkey?

Also please use snake case instead camel case on schemas properties

Reply:
I will use it in the next part of the PR.
When an offer is taken the bot will tag the user's npub before publishing the event.